src/entries/web-runtime-with-compiler.js

```diff
@@ -26,7 +26,10 @@ Vue.prototype.$mount = function (el: string | Element): Component {
       } else if (template.nodeType) {
         template = template.innerHTML
       } else {
-        warn('invalid template option:' + template, this)
+        if (process.env.NODE_ENV !== 'production') {
+          warn('invalid template option:' + template, this)
+        }
+        return mount.call(this, el)
```

Comment:
if specify the invalid template, there is a possibility that runtime error in render process.
I don't know whether this approach is correct.